dev/standalone/Dockerfile

```diff
@@ -11,6 +11,9 @@ RUN set -ex; \
 
 USER root
 
-RUN dnf install -y gettext
+RUN dnf install -y gettext; \
```

Comment:
Maybe it can go in https://github.com/ansible/galaxy_ng/blob/master/dev/Dockerfile.base ?

So it is reused by insights, keycloak and other images?